alternator/executor.cc

```diff
@@ -927,9 +928,10 @@ static future<executor::request_return_type> create_table_on_shard0(tracing::tra
             if  (!range_key.empty() && range_key != view_hash_key && range_key != view_range_key) {
                 add_column(view_builder, range_key, attribute_definitions, column_kind::clustering_key);
             }
-            sstring where_clause = "\"" + view_hash_key + "\" IS NOT NULL";
+            sstring where_clause = format("{} IS NOT NULL", cql3::util::maybe_quote(view_hash_key));
```

Comment:
I think the problem you found here can actually cause a real bug - please try a test where the key's name contains the double-quote character (I think that's legal?). In that case it's not enough to add quotes before and after the key name - you actually need to double the double-quote. So you discovered a real bug.

So you're write that cql3::util::maybe_quote(), or cql3::util::quote(), is the right thing to call (I didn't understand your mention of the "describe table" and why you prefered maybe_quote() over quote()).